pkg/model/test.go

```diff
@@ -41,6 +41,7 @@ var (
 	ErrHostMalformed   = fmt.Errorf("host is malformed")
 	ErrInvalidHostName = fmt.Errorf("invalid hostname")
 	ErrInvalidIp       = fmt.Errorf("invalid ip")
+	ErrEmptyCommands   = fmt.Errorf("commands cannot be empty")
```

Comment:
Could we include the test name in the error? I think it would help to users to know exactly where the problem might be